packages/components/src/Table/Row/TableRow.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TableCell from '../Cell/TableCell';
+import theme from './TableRow.scss';
+
+export function getRowId(rowDataGetter, element, index) {
+	if (rowDataGetter && rowDataGetter.getElementId) {
+		return rowDataGetter.getElementId(element);
+	} else if (element.id && typeof element.id === 'string') {
+		return element.id;
+	}
+	return index.toString();
+}
+
+function getRowClassName(classnames, index) {
+	if (
+		classnames &&
+		classnames.rows &&
+		Array.isArray(classnames.rows) &&
+		classnames.rows.length >= index
+	) {
+		return classnames.rows[index];
+	}
+	return null;
+}
+
+function getRowDataClassName(column) {
```

Comment:
This function is not necessary. You can directly write
```
column.cellClassName || `tc-table-row-data-${column.key}`
```

But shouldn't you add both classnames ?